core/lib/spree/release_drafter.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
```

Comment:
This set of files seems completely detached from solidus shipped code, can we find a better place for it?
I guess the problem could be where to put the specs… but it also looks like they don't need any special setup and can be run independently, so maybe an extra job in circleci would be fine.

I don't have great locations to suggest, one would be a subfolder of bin/, or of .github/workflows/, or maybe a new top-level folder?

Reply:
I also felt that there was something off in that regard. I didn't want to do a complex setup only for that, but it probably makes sense. As you said, nothing needs to be shipped to the user. That we need to branch on the ruby version is also a smell of the loose cohesion.

I'll investigate creating a new `tools` top directory with its own Gemfile. This way, we won't need to branch on the main Gemfile either.

Reply:
We discussed it in the Core Team meeting and decided to give it more thought. We commented on different options:

- Top `tools` directory.
- Folder within `bin`.
- Separate gem
- Part of solidus_dev_support

For now, I've put it within a top `tools` directory for a better structure in the review. However, I still haven't added the CircleCI job, which means that the tests are not running now—putting the PR as a draft until we decide.

Reply:
I've gone with a dedicated `dev_tools` folder. We're going to need more code for further automation, so I think it's good to have a proper structure for it. Going for a gem or making it part of soliidus_dev_support looks overkill at this point. Do you agree @elia, @kennyadsl?

Reply:
@kennyadsl, about your concern about a `dev_tools` being confusing for contributors, I've added a README file within with an explicit description. Let me know if that would address your concern.

Reply:
`solidus_dev_support` is meant to help supporting multiple solidus versions and not well suited for this, an external gem is definitely overkill 💯.

It's good anyway that this has no consequence on versions and we're free to adjust as we go, so I'm not too concerned with the name or structure 🙌.

Reply:
@waiting-for-dev works for me, thanks for adding the README. I think we also want to exclude this folder from the files we ship in the gem via the gemspec, right?